### PR TITLE
testing/flann: add aport

### PR DESCRIPTION
**Pull request by chambbj**

### CODE REVIEW

File: testing/flann/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=flann
+pkgver=1.9.1
+pkgrel=0
+pkgdesc="Fast Library for Approximate Nearest Neighbors"
+url="http://people.cs.ubc.ca/~mariusm/flann"
+arch="all"
+license="BSD"
+options="!check"
```
Comment (TBK) [inline testing/flann/APKBUILD]: Please write a short comment why the check is disabled.

File: testing/flann/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+depends=""
+makedepends="cmake"
+install=""
```
Comment (TBK) [inline testing/flann/APKBUILD]: Please remove `install=""` when not used.

**Comment by chambbj** [comment]
@TBK I tried enabling the tests, but they appear to be a mess upstream (pull in extra dependencies, broken links to test data downloads, etc.). I added a note to that effect, and will file a ticket to hopefully have the issue resolved in a future release. Would like to proceed without tests for now though, if possible.

**Comment by TBK** [comment]
If you squash the commits and name it "testing/flann: add aport" then it should be good to go.

**Comment by chambbj** [comment]
Thanks @TBK. Done.

**Comment by chambbj** [comment]
@TBK ping. Any chance we can get this merged? It's a dependency of another package I'm working one (PCL) that would feed into the PDAL package you just reviewed in #3933.

File: testing/flann/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=flann
+pkgver=1.9.1
+pkgrel=0
+pkgdesc="Fast Library for Approximate Nearest Neighbors"
+url="http://people.cs.ubc.ca/~mariusm/flann"
+arch="all"
+license="BSD"
```
Comment (BernhardGruen) [inline testing/flann/APKBUILD]: Please see https://spdx.org/licenses/

**Comment by chambbj** [comment]
@fabled I wonder if you may have a chance to look at this PR. Should be ready to go. Thanks!

**Comment by clandmeter** [comment]
What does this cmake patch do, and has it been reported upstream?

**Comment by chambbj** [comment]
@clandmeter It has been reported [here](https://github.com/mariusmuja/flann/issues/369) with a slightly different fix suggested by the author than what I implemented. Problem is that `add_library` behaves a little differently on CMake v3.11 and the library has not been updated yet to handle this.

**Comment by clandmeter** [comment]
ok good, next time add that info to the PR so we know whats it about.
We don't always have time to debug patches :)

Ill commit it as is now but as you are the maintainer we expect you to keep an eye on it.

Thanks!

**Comment by chambbj** [comment]
Very good. Thanks!

**Pull request closed without merging**